edge/pkg/edged/cadvisor/cadvisor_linux.go

```diff
@@ -0,0 +1,93 @@
+// +build cgo,linux
+
+/*
+Copyright 2015 The Kubernetes Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
```

Comment:
Is this file modified or introduced as is? Please add description about this.

Reply:
done

Reply:
add the description in detail like other introduced file